app/assets/stylesheets/application.scss

```diff
@@ -2,3 +2,14 @@
 *= require foundation_and_overrides
 *= require font-awesome
 */
+nav {
```

Comment:
This should be in its own file. The `application.scss` file should just require other stylesheets, each of which should have its own file